applications/notes/HM.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+<body>
+<header>
+    <section>
+        <h1> 
+            Notedly
+
+        </h1>
+    </section>
```

Comment:
You should close your `<header>` at this point. It shouldn't include the main content of the page.